crates/uv-resolver/src/lock/mod.rs

```diff
@@ -5047,21 +5047,15 @@ impl std::fmt::Display for WheelTagHint {
                     };
                     writeln!(
                         f,
-                        "{}{} You're on {}, but there are no wheels for the current platform, consider configuring `{}`.",
+                        "{}{} You're on {}, but {} only has wheels for the following platform{s}: {}; consider adding your platform to `{}` to ensure uv resolves to a version with compatible wheels",
                         "hint".bold().cyan(),
                         ":".bold(),
                         best,
-                        "tool.uv.required-environments".green()
-                    )?;
-                    write!(
```

Comment:
I'm willing to have two messages still, but if we do I think it should be in this order, i.e.,

hint: There are only wheels for the following platforms ..
hint: Consider adding your platform to ...

I interested in seeing how a single hint feels too though

Reply:
The single path is def nicer.